sql/2019/13_Ecommerce/13_13.sql

```diff
@@ -0,0 +1,18 @@
+#standardSQL
+# 13_13: Lighthouse PWA scores
+SELECT
+  JSON_EXTRACT_SCALAR(report, "$.audits.categories.pwa.score") AS pwa_score,
```

Comment:
Getting NULLs here. Did you mean `$.categories.pwa.score`?

Reply:
Good catch. Fixed.